src/gausshermite.jl

```diff
@@ -1,12 +1,18 @@
 @doc raw"""
-    gausshermite(n::Integer) -> x, w  # nodes, weights
+    gausshermite(n::Integer; probabilist = false) -> x, w  # nodes, weights
```

Comment:
```suggestion
    gausshermite(n::Integer; normalize = false) -> x, w  # nodes, weights
```
Maybe, some other name of the keyword argument would be better?

Reply:
I've made your suggested changes and added tests.